fabtests/pytest/efa/test_rdm.py

```diff
@@ -23,18 +23,42 @@ def run_rdm_test(cmdline_args, executable, iteration_type, completion_type, memo
                           pytest.param("standard", marks=pytest.mark.standard)])
 def test_rdm_pingpong(cmdline_args, iteration_type, completion_type, memory_type):
     run_rdm_test(cmdline_args, "fi_rdm_pingpong", iteration_type,
-                 completion_type, memory_type)
+            completion_type, memory_type, "all")
+
+@pytest.mark.parametrize("message_size",
+                         [pytest.param("r:8000,4,9000", marks=pytest.mark.functional),
+                          pytest.param("r:0,4,64", marks=pytest.mark.functional),
+                          pytest.param("r:4048,4,4148", marks=pytest.mark.functional)])
```

Comment:
Since this range are used repeatedly, we can define a fixture for it.

See line 9 `memory_type` for example.

An minor issue is that when you try to use the new fixture in `test_rma_bw`, you will need to explicitly import it.

So I suggest to create a new file `efa_common.py` and put the new fixture and `memory_type` there.

Reply:
Parameterize is essentially doing the same thing here. I don’t see the value in moving to fixtures, it adds further complexity IMO.

Reply:
It is the similar, but have it as fixture do remove duplicated code, and we might want to add more ranges in the future.

Reply:
Shared fixtures have to be declared in conftest.py and I moved them to conftest as recommended in pytest documentation. However, our conftest.py is global to pytest and in the long run, we should think about having plug-ins to import fixtures from individual modules.